configuration/security.md

```diff
@@ -76,21 +76,21 @@ sudo deluser -remove-home pi
 
 ## Make `sudo` require a password
 
-Placing `sudo` in front of a command runs it as a superuser, and by default, that does not need a password. In general, this is not a problem. However, if your Pi is exposed to the internet and somehow becomes exploited (perhaps via a webpage exploit for example), the attacker will be able to change things that require superuser credential, unless you have set `sudo` to require a password.
+Placing `sudo` in front of a command runs it as a superuser, and by default, that does not need a password. In general, this is not a problem. However, if your Pi is exposed to the internet and somehow becomes exploited (perhaps via a webpage exploit for example), the attacker will be able to change things that require superuser credentials, unless you have set `sudo` to require a password.
 
 To force `sudo` to require a password, enter:
 
 ```bash
-sudo nano /etc/sudoers.d/010_pi-nopasswd
+sudo visudo /etc/sudoers.d/010_pi-nopasswd
 ```
 
 and change the `pi` entry (or whichever usernames have superuser rights) to:
 
 ```bash
-alice ALL=(ALL) PASSWD: ALL
+pi ALL=(ALL) PASSWD: ALL
 ```
 
-Now save the file.
+Then save the file: it will be checked for any syntax errors. If not errors were detected, the file will be saved and you will be returned to the shell prompt (`username@host:directory $`). If errors were detected, you will be asked 'what now?' - press enter/return and it will show a list of options. You will probably want to use '(e)dit sudoers file again,' so you can go in and see what the problem is.
```

Comment:
Now that you've got this more detailed explanation of `visudo` (nice :+1: ) perhaps it's worth linking to here from the other pages where visudo is mentioned?

Reply:
I've copied that paragraph into linux/usage/users.md, which is the only other explanation of how to use visudo.

Reply:
I hate seeing duplication in the documentation, as when one location gets improved / fixed later the other location(s) inevitably get missed :confused: 
Although I guess the `visudo` command isn't going to be changing anytime soon, so perhaps that can be ignored in this case?